internal/platform/utimes.go

```diff
@@ -0,0 +1,31 @@
+package platform
+
+import (
+	"io/fs"
+	"syscall"
+)
+
+// UtimesNano is like syscall.UtimesNano. This returns syscall.ENOENT if the
+// path doesn't exist.
+//
+// See https://linux.die.net/man/3/futimens
```

Comment:
ps I'm thinking we change to posix for links like this instead of man, since we're often told to look at posix when in doubt. wdyt @evacchi @mathetake https://pubs.opengroup.org/onlinepubs/9699919799/functions/futimens.html